bedrock/exp/templates/exp/firefox/whatsnew/whatsnew-fx70.html

```diff
@@ -1,115 +0,0 @@
-{# This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
I'm not entirely clear as the issue doesn't mention this, but why is this PR removing this template?

Reply:
I think that was my own misunderstanding. Adjusting PR. :+1: